db/migrate/20170103085726_create_cashboxes.rb

```diff
@@ -0,0 +1,9 @@
+class CreateCashboxes < ActiveRecord::Migration
+  def change
+    create_table :cashboxes do |t|
+      t.integer :balance, default: 0
```

Comment:
For the moment let's  have  amount on payment/transaction as we agreed the first time to move forward with the feature.